src/python/WMCore/WMSpec/Steps/Executors/DQMUpload.py

```diff
@@ -50,6 +56,14 @@ class DQMUpload(Executor):
 
     """
 
+    def __init__(self):
+        super(DQMUpload, self).__init__()
+        self.retryDelay = 300
+        self.retryCount = 3
+        self.registerLFNBase = '/store/unmerged/DQMGUI'
+        self.registerEOSPrefix = '/eos/cms'
+        self.registerURL = 'https://cmsweb-testbed.cern.ch/dqm/offline-test-new/api/v1/register'
```

Comment:
Can we derive this URL from the url used for uploading the root file to the DQM Gui?

Reply:
Do you meant `step.upload.URL`? Aren't they independent?